scripts/translations/sort_filter.jq

```diff
@@ -0,0 +1,13 @@
+def walk(f):
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-18T16:52:16Z" title="Tuesday, February 18th 2020, 5:52:16 pm +01:00">Feb 18, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,13 @@
+def walk(f):
```


Whatever that file is doing :laughing:

Reply:
> [<img alt="Mogge" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2020-02-18T16:56:58Z" title="Tuesday, February 18th 2020, 5:56:58 pm +01:00">Feb 18, 2020</time>_

---


Wonders